ide/gsf.testrunner.ui/src/org/netbeans/modules/gsf/testrunner/ui/api/TestMethodController.java

```diff
@@ -36,16 +36,22 @@ public static void setTestMethods(Document doc, List<TestMethod> methods) {
     }
 
     public static final class TestMethod {
+        private final String testClassName;
         private final SingleMethod method;
         private final Position start;
         private final Position end;
 
-        public TestMethod(SingleMethod method, Position start, Position end) {
+        public TestMethod(String testClassName, SingleMethod method, Position start, Position end) {
```

Comment:
Looks like this is causing a [sigtest failure](https://github.com/apache/netbeans/pull/2788#issuecomment-791267713) - probably return the original constructor back as part of #2788